### PR TITLE
Create AnushaSP07.json

### PR DESCRIPTION
**Pull request by AnushaSP07**
data: AnushaSP07 - committing Anusha SP profile link details to LinkFree

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4645"></a>

### CODE REVIEW

File: data/AnushaSP07.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    },
+     {
+      "name": "Follow me on Medium",
+      "url": "https://medium.com/@anushasp07",
+      "icon": "FaMedium"
+    },
+    {
+      "name": "Follow me on HashNode",
+      "url": "https://hashnode.com/@AnushaSP07",
+      "icon": "FaHashnode"
```
Comment (priyankarpal) [inline data/AnushaSP07.json:20]: Change the icon from`FaHashnode` to `SiHashnode` 
``` suggestion
      "icon": "SiHashnode"
```

File: data/AnushaSP07.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    },
+    {
+      "name": "Connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/anushasp07/",
+      "icon": "FaLinkedIN"
```
Comment (priyankarpal) [inline data/AnushaSP07.json:25]: Also, change the `FaLinkedIN` to `SiLinkedin`
``` suggestion
      "icon": "SiLinkedin"
```

Comment (AnushaSP07) [reply data/AnushaSP07.json:25]: Thank you @priyankarpal , sure I will make changes and commit

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/AnushaSP07.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Anusha SP",
+  "displayStatsPublic": true,
```
Comment (kumarsonsoff3) [inline data/AnushaSP07.json:3]: As of our latest update, we no longer need the `displayStatsPublic` property. So please remove this property from your json. 

```suggestion
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

**Review by Panquesito7 — APPROVED** [review]

**Review by priyankarpal — APPROVED** [review]
Looking good to me.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉

**Pull request merged**